frontend/src/components/sidebar/sidebar.module.scss

```diff
@@ -2,6 +2,7 @@
 
 .container {
   background-color: $colour-darkBlue;
+  grid-area: sidebar;
```

Comment:
Could you explain the reasoning to use grid area? Rather than wrapping the sidebar in a div with a width from the outside. It feels like doing it this way makes, styling the component implementation dependent? In that you have to know internal details to use it

Reply:
That's how grid is used in general, the parent components says where each child component should be placed on the grid and child components are identified by name.

I don't think of css as an implementation detail. Any user can preview the element's style in the browser and any css can always be overridden.
The only question is where to put the child component's grid name css, which could be either with the component or with home.

The upside of using grid is that you don't need any extra divs just to get the layout right, which helps to keep the html clean.

Reply:
Will it be straightforward to hide the sidebar on a narrow window with grid?

Reply:
I haven't tried, but you should be able to by wrapping it inside `@media` block like shown
https://webmadewell.com/css-grid-responsive-layouts/